install/kubernetes/cilium/values.yaml

```diff
@@ -648,6 +648,10 @@ hubble:
     server:
       cert: ""
       key: ""
+      # -- extra DNS names added to certificate when its auto gen
```

Comment:
tiny nit:

```suggestion
      # -- Extra DNS names added to certificate when it's auto genenrated
```

And similarly below.

Reply:
^ Note that the above still needs another fix for `generated` spelling :sweat_smile:

Reply:
Fixed :sweat_smile: